docs/api-overview.md

```
@@ -3023,6 +3025,119 @@ _Appears in:_
+ServiceMesh is the Schema for the servicemesh API
+
+
+
+_Appears in:_
+- [ServiceMeshList](#servicemeshlist)
+
+| Field | Description | Default | Validation |
+| --- | --- | --- | --- |
+| `apiVersion` _string_ | `services.platform.opendatahub.io/v1alpha1` | | |
+| `kind` _string_ | `ServiceMesh` | | |
+| `kind` _string_ | Kind is a string value representing the REST resource this object represents.<br />Servers may infer this from the endpoint the client submits requests to.<br />Cannot be updated.<br />In CamelCase.<br />More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds |  |  |
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)